src/splits.rs

```diff
@@ -54,6 +55,10 @@ pub enum Split {
     ///
     /// Splits when player HP is 0
     PlayerDeath,
+    /// Mask Shard Obtain (Event)
+    ///
+    /// Splits when the player obtains a new mask shard
+    MaskShardObtain,
```

Comment:
People who are used to the Hollow Knight autosplitter's conventions will be expecting something more like:
```
/// Mask Shard (Obtain)
///
/// Splits when obtaining a Mask Shard or a complete Mask upgrade
OnObtainMaskShard,
```
And as mentioned below, the "or a complete Mask" part is important since it should only split on completing that mask and not on obtaining the shard that's about to complete it